Software/tft/header/config.go

```diff
@@ -0,0 +1,68 @@
+package header
+
+import (
+	"errors"
+	"os"
+)
+
+type Config struct {
+	Apply            bool
+	InputPath        string
+	OutputPath       string
+	VendorId         int
+	DeviceId         int
+	HardwareRevision int
+	InputFile        *os.File
+	OutputFile       *os.File
+}
+
+// Check config structure for valid values
+// PCI Vendor ID, Device ID should fit into uint16 and be positive
+// PCI Hardware Revision should be any uint16
+func CheckConfig(c *Config) error {
```

Comment:
Correct way (to generate godoc automatically) is to start a comment with the method name:
> CheckConfig checks config structure...